loleaflet/src/layer/tile/CommentSection.ts

```diff
@@ -71,6 +82,542 @@ class CommentSection {
 
 	}
 
+	private adjustComment (comment: any) {
+		var rectangles, color, viewId;
+		comment.trackchange = false;
+		rectangles = L.PolyUtil.rectanglesToPolygons(L.LOUtil.stringToRectangles(comment.textRange || comment.anchorPos), this.map._docLayer);
```

Comment:
We are trying to get rid of L.Poly*, not urgent but please use CPoly* in future.